
# Articles Summary

## PICRUSt2 
 The PICRUSt2 method now consists of phylogenetic placement, hidden-state-prediction, and sample-wise gene abundance tabulation. ASV sequences and abundances
are taken as input, and stratified gene family and pathway abundances are output.


## 16s RNA
The most common approach for profiling communities is to sequence the highly conserved 16S rRNA
gene (16S), which acts as an identifier that may be enumerated to determine the relative
abundances of prokaryotic groups present.

Functional profiles cannot be directly identified from 16S sequence data due to strain variation and the lack of uniqueness of 16S rRNA gene
sequences among microbes, but several approaches have been developed to infer approximate
microbial community functions from taxonomic profiles (and thus amplicon sequences) alone;
Importantly, these methods predict potential functions encoded at the level of DNA

16S rRNA gene sequencing  , which profile selected organisms or single marker genes, are sometimes referred
to as metagenomics, but this is a misnomer, as they do not target the
entire genomic content of a sample.

## Denoising methods 
They enable sequence resolution down to the single-nucleotide level. This improved resolution allows closely
related organisms to be better distinguished and thus more precise gene annotations are
associated with a given 16S sequence.

## Shotgun analysis
One such method, shotgun metagenomics, is the untargeted (‘shotgun’) sequencing of all (‘meta-’) microbial genomes
‘genomics’ present in a sample.
A typical shotgun metagenomics study comprises five steps, after the initial study design:

  1.  the collection, processing and sequencing of the samples; 
  1.  preprocessing of the sequencing reads; 
  1.  sequence analysis to profile taxonomic, functional and genomic fea- tures of the microbiome; 
  1.  statistical and biological post-processing analysis, and;
  1.  validation